cp23/hpi-fun2/cubes/executables/generator.cpp

```diff
@@ -0,0 +1,79 @@
+#include <testlib.h>
+#include <bits/stdc++.h>
+
+using namespace std;
+
+const string_view SAMPLE1 = R"(2
+1 1 1
+1 1 2
+)";
```

Comment:
You may want to add another (a bit larger) sample.

Maybe one that shows that your cloud can be disconnected if this is the case.